adafruit_max31865.py

```diff
@@ -96,16 +96,24 @@ class MAX31865:
     # thread safe!
     _BUFFER = bytearray(3)
 
-    def __init__(self, spi, cs, *, rtd_nominal=100, ref_resistor=430.0, wires=2):
+    def __init__(self, spi, cs, *, rtd_nominal=100, ref_resistor=430.0, wires=2, filter_freq=60):
```

Comment:
Please make it `filter_frequency` because it'll be easier to understand who may not know what `freq` is short for.

Reply:
No problem. Should be good now.